pkg/entitlements/subscriptions.go

```diff
@@ -0,0 +1,155 @@
+package entitlements
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/stripe/stripe-go/v81"
+)
+
+// CreateSubscription creates a new subscription
+func (sc *StripeClient) CreateSubscription(params *stripe.SubscriptionParams) (*stripe.Subscription, error) {
+	subscription, err := sc.Client.Subscriptions.New(params)
+	if err != nil {
+		return nil, err
+	}
+
+	return subscription, nil
+}
+
+// ListStripeSubscriptions lists stripe subscriptions by customer
+func (sc *StripeClient) ListOrCreateStripeSubscriptions(customerID string) (*Subscription, error) {
```

Comment:
We should be consistent on the name, most of these functions don't say `Stripe` except this one. This should follow that 
```suggestion
func (sc *StripeClient) ListOrCreateSubscriptions(customerID string) (*Subscription, error) {
```